packages/fxa-auth-server/config/index.js

```diff
@@ -1206,10 +1218,8 @@ conf.set(
 );
 conf.set('smtp.verifyPrimaryEmailUrl', `${baseUri}/verify_primary_email`);
 conf.set('smtp.verifySecondaryEmailUrl', `${baseUri}/verify_secondary_email`);
-conf.set('smtp.subscriptionDownloadUrl', `${baseUri}/subscriptions/download`);
```

Comment:
Not sure why we removed the config var and built this URL in code with a refactor in 330a4926dae26fbe0c845b40a36329290d334fbe - but this is why the env var with FPN URL wasn't being used. There's no `/subscriptions/download` route on content-server. I suppose as part of #2026 we could make one in the future that picks the correct redirect URL based on subscription plan and product, but we don't have one right now.